boot/boot.go

```diff
@@ -41,13 +41,23 @@ const (
 	TryingStatus = "trying"
 )
 
+// RebootInfo contains information about how to perform a reboot if
+// required
+type RebootInfo struct {
+	// rebootRequired is true if we need to reboot after an update
+	RebootRequired bool
+	// Rbl will not be nil if the bootloader needs some control on
+	// how to perform the rebot
```

Comment:
Looking at the code, it appears to me that this is not really a need, but rather an interface offering more control. So maybe this should be formulated like "Rbl will be set if the bootloader offers some control on how to perform the reboot", unless I misinterpreted something?

Reply:
Well, to me is that `bootloader` needs to say something about how to reboot to the `boot` package, as it is not actually offering extra functionality. It would be more control if it implemented itself the Reboot call. Anyway, maybe the wording is a bit misleading, I'll refactor it a bit.